app/views/browse/service-instance.html

```diff
@@ -0,0 +1,102 @@
+<project-page>
+
+  <!-- Middle section -->
+  <div class="middle-section">
+    <div class="middle-container">
+      <div class="middle-header">
+        <div class="container-fluid">
+          <breadcrumbs breadcrumbs="breadcrumbs"></breadcrumbs>
+          <alerts alerts="alerts"></alerts>
+          <div ng-if="!loaded" class="mar-top-xl">Loading...</div>
+          <div ng-if="serviceInstance">
+            <h1 class="contains-actions">
+              <div class="pull-right dropdown" ng-hide="!('instances' | canIDoAny)">
+                <button type="button" class="dropdown-toggle btn btn-default actions-dropdown-btn hidden-xs" data-toggle="dropdown">
+                  Actions
+                  <span class="caret"></span>
+                </button>
+                <a href=""
+                   class="dropdown-toggle actions-dropdown-kebab visible-xs-inline"
+                   data-toggle="dropdown"><i class="fa fa-ellipsis-v"></i><span class="sr-only">Actions</span></a>
+                <ul class="dropdown-menu dropdown-menu-right actions action-button">
+                  <li ng-if="{resource: 'instances', group: 'servicecatalog.k8s.io'} | canI : 'update'">
+                    <a ng-href="{{serviceInstance | editYamlURL}}" role="button">Edit YAML</a>
+                  </li>
+                  <li ng-if="{resource: 'instances', group: 'servicecatalog.k8s.io'} | canI : 'delete'">
+                    <delete-link
+                      kind="Instance"
+                      resource-name="{{serviceInstance.metadata.name}}"
+                      project-name="{{serviceInstance.metadata.namespace}}"
+                      alerts="alerts">
+                    </delete-link>
+                  </li>
+                </ul>
+              </div>
+              {{serviceInstance | serviceInstanceDisplayName:serviceClasses}}
+              <small class="meta">created <span am-time-ago="serviceInstance.metadata.creationTimestamp"></span></small>
+            </h1>
+            <labels labels="serviceInstance.metadata.labels" clickable="true" kind="instance" project-name="{{serviceInstance.metadata.namespace}}" limit="3"></labels>
+          </div>
+        </div>
+      </div><!-- /middle-header-->
+      <div class="middle-content" persist-tab-state>
+        <div class="container-fluid">
+          <div class="row" ng-if="serviceInstance">
+            <div class="col-md-12">
+              <uib-tabset>
+                <uib-tab active="selectedTab.details">
+                  <uib-tab-heading>Details</uib-tab-heading>
+                  <div class="resource-details">
+                    <div class="row">
+                      <div class="col-lg-6">
+                        <h3>Plan</h3>
+                        <p>
+                          {{plan.description}}
+                        </p>
+                        <dl class="dl-horizontal left">
+                          <dt ng-if-start="serviceClass.description">Description:</dt>
```

Comment:
We should show `externalMetadata.longDescription` if it's there even if `description` is not.